### ISSUE TITLE
Need to Discuss how to best show monthly quota for plant. Do we do in App or do we build report in hub and email it 

### ISSUE THREAD

**Issue by johnmurphy01** [gatewayapps/ims-feedback#499]
Version: 0.11.0 

 User: Jim Byrd(jim.byrd@nucor-yamato.com)

**Comment by jimbyrd** [comment]
![image](https://user-images.githubusercontent.com/28160043/124814664-e98f4300-df2b-11eb-9bce-ee8dac51ebbd.png)

**Comment by jimbyrd** [comment]
Or if we could do a export to excel from 'Observations by Reporter Area"

**Comment by johnmurphy01** [comment]
Does each department have a different quota?

**Comment by jimbyrd** [comment]
Basically, each supervisor is required 1 per week. So, 4 week month, quota is 4, but….. a supervisor can delegate to a lead or someone to do one of his, which makes it tricky

**Comment by johnmurphy01** [comment]
Yeah, the only way I see that being possible is if when you submit an observation, you can select a person for whom you are submitting. So if I were entering an observation for you, I could select you and the observation would count toward your quota.

Are supervisors the only ones with quotas? We could show their quota number for the month somewhere in the app. Or provide a new chart that displays this

**Comment by jimbyrd** [comment]
Right now, I think we just focus on reporting the 1 per supv, the delegating would be tough
